gatsby-node.js

```diff
@@ -150,3 +180,18 @@ exports.createResolvers = ({ createResolvers }) => {
   }
   createResolvers(resolvers)
 }
+
+exports.onPostBootstrap = () => {
+    var index = eJson('public/search_index.json');
+    var images = eJson('public/imageSrc.json').data.data.allStrapiProject.edges;
+
+    // en -> store
+    for (var i = 0; i < images.length; i++) {
+      var node = images[i].node;
+      var id = node.id;
+      var image = node.project_image.childImageSharp.fluid.src;
```

Comment:
Ah-ha! Here's the problem. We need to save the _entire_ `childImageSharp` object, not just the `src` part, so we can pass the entire `sercSet` plus the width/aspectRatio/etc. to the gatsby-image plugin.